Gems/Prefab/PrefabBuilder/PrefabGroup/tests/PrefabBehaviorTests.cpp

```diff
@@ -331,15 +331,21 @@ namespace UnitTest
         auto scene = CreateEmptyMockSceneWithRoot("Manifest", "//o3de/watch.folder/manifest_src_file.xml", "//o3de/watch.folder");
 #endif
         /*---------------------------------------\
+        Notes on the graph hierarchy:
+        - 3m is a node with mesh data. Proc prefab builder will create an entity for this node
+        - 4t contains the transform data for the mesh. It's a child of 3m because a node can only contain one data item.
```

Comment:
> '.... because a node can only contain one data item'

Is this limitation coming from the fbx file format or a design choice made while building the nodes? I ask because this approach of making the first child contain transform data seems counterintuitive to me. Ideally if the mesh node itself can also hold the transform data, it'll be an easier mental model to work through and also potentially won't cause node-sync issues if the tree gets modified later.

Reply:
It's how the SceneGraph class stores content. I don't know the decision process on this but perhaps @AMZN-stankowi or @jackalbe can give more info.